src/main/java/com/google/api/generator/engine/writer/JavaWriterVisitor.java

```diff
@@ -351,6 +356,31 @@ public void visit(TryCatchStatement tryCatchStatement) {
     newline();
   }
 
+  /** =============================== COMMENT =============================== */
+  public void visit(LineComment lineComment){
+      // Split comments by new line and add `//` to each line.
+      String formattedSource =  JavaFormatter.format(String.format("// %s", String.join("\n//", lineComment.comment().split("\\r?\\n"))));
+      buffer.append(formattedSource);
+  }
+
+  public void visit(BlockComment blockComment){
+    // Split comments by new line and embrace the comment block with `/** */`.
+    String sourceString = blockComment.comment();
+    String formattedSource =  JavaFormatter.format(String.format("%s %s %s", BLOCK_COMMENT_START, sourceString, BLOCK_COMMENT_END));
+    buffer.append(formattedSource);
+  }
+
+  public void visit(JavaDocComment javaDocComment){
+    StringBuilder formattedJavaDocComment = new StringBuilder();
+    formattedJavaDocComment.append(BLOCK_COMMENT_START + NEWLINE);
```

Comment:
Please append BLOCK_COMMENT_START and NEWLINE separately.

Reply:
done. thanks!